biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -423,16 +409,24 @@ subroutine FatesPlantRespPhotosynthDrive (nsites, sites,bc_in,bc_out,dtime)
                   ! scratch space.
                   ! ------------------------------------------------------------------------
                   rate_mask_z(:,1:numpft,:) = .false.
+                  psn_z(:,:,:)     = 0._r8
+                  anet_av_z(:,:,:) = 0._r8
+                  c13disc_z(:,:,:) = 0._r8
+                  rs_z(:,:,:)      = 0._r8
+                  lmr_z(:,:,:)     = 0._r8
+                                    
+                  if_any_cohorts: if(currentPatch%countcohorts > 0.0)then
```

Comment:
This is returning a build error on `derecho`.  I think this is because`countcohorts` is a subroutine, not a function.

Reply:
weird, wonder why it compiled for me, maybe this fail was with gnu? Anyway, I updated this to use the output of countcohorts:  patch%num_cohorts.